web/app/js/components/Endpoints.jsx

```diff
@@ -0,0 +1,110 @@
+import BaseTable from './BaseTable.jsx';
+import ErrorBanner from './ErrorBanner.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Spinner from './util/Spinner.jsx';
+import _each from 'lodash/each';
+import _get from 'lodash/get';
+import _reduce from 'lodash/reduce';
+import { apiErrorPropType } from './util/ApiHelpers.jsx';
+import { numericSort } from './util/Utils.js';
+import { withContext } from './util/AppContext.jsx';
+import withREST from './util/withREST.jsx';
+
+const endpointColumns = [
+  {
+    title: "Namespace",
+    dataIndex: "namespace",
+    sorter: (a, b) => (a.namespace).localeCompare(b.namespace)
+  },
+  {
+    title: "IP",
+    dataIndex: "pod.podIP"
+  },
+  {
+    title: "Port",
+    dataIndex: "addr.port",
+    sorter: (a, b) => numericSort(a.addr.port, b.addr.port)
+  },
+  {
+    title: "Pod",
+    dataIndex: "name",
+    sorter: (a, b) => (a.name).localeCompare(b.name)
+  },
+  {
+    title: "Resource Version",
+    dataIndex: "pod.resourceVersion"
```

Comment:
tioli perhaps sorting by resource version could be useful in determining pod age / service discovery staleness?

Reply:
ohhh that's interesting. I'll add it!